lib/gui/pages/settings/templates/settings.tpl.html

```diff
@@ -7,7 +7,7 @@ <h1 class="title space-bottom-large">Settings</h1>
         ng-model="settings.currentData.errorReporting"
         ng-change="settings.toggle('errorReporting')">
 
-      <span>Report errors</span>
+      <span>Report errors back to resin.io</span>
```

Comment:
How about "Enable sending error reports [to resin.io]"? That'd be a similar wording to what browsers and other programs use, and the "sending" should make it clear that there's stuff going over the wire.

Maybe we could also split it up into error reports & telemetry (just a thought, but that's out of the scope of this PR)?